administrator/components/com_admin/script.php

```diff
@@ -2252,6 +2254,8 @@ public function postflight($action, $installer)
             return false;
         }
 
+        $this->setGuidedToursUid();
```

Comment:
```suggestion
        $this->setGuidedToursUid();
```
The method returns ture if success and false if error. Why don't you check the result and return false here like it is done with the other called function?

Reply:
I didn't think it was important enough to cause the Joomla update to fail if the tours uid could not be auto-set.

I can remove the return value.

Reply:
Will guided tours work after an update from a previous 5.0.0-alpha version if this method fails or is not called at all? If yes, then it's ok as it is. But if guided tours will not work in this case, we should check the return code and return false when it has returned false.

Reply:
Yes, guided tours will work if it fails.